ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -975,7 +976,7 @@ open class DrawerController: UIViewController, FluentUIWindowProvider {
 
 extension DrawerController {
     static var drawerBackgroundColor: UIColor {
-        return MSFDrawerTokens().drawerHorizontalContentBackground
+        return MSFDrawerTokens().drawerContentBackground
```

Comment:
Do you have any cleanup that needs to be done in the LeftNav Demo controller?

Reply:
The color discrepancy in left nav demo controller and left nav devmain are different scenarios. 

The left nav demo controller it stems from inconsistent tokens in list [background](https://github.com/microsoft/fluentui-apple/blob/99b4f3b141cd36cdba3512f68592353d79b6d2f3/tools/sgen/input/FluentUIStyle.yml#L313)  and drawer [background](https://github.com/microsoft/fluentui-apple/blob/99b4f3b141cd36cdba3512f68592353d79b6d2f3/tools/sgen/input/FluentUIStyle.yml#L286) 

The overrides in demo controller essentially sets a transparent color to list (via client override) to immerse in left nav. List when used in isolation have different token values in Figma file and is expected behavior.